test_pytest_rerunfailures.py

```diff
@@ -219,3 +221,32 @@ def test_fail():
                                '--result-log', './pytest.log')
 
     assert_outcomes(result, passed=0, failed=1, rerun=2)
+
+
+@pytest.mark.parametrize('delay_time', [0, 0.0, 1, 2.5])
```

Comment:
I like the use of  the parametrize very much. As this is now easily possible I would like to test here also the unhappy path of a negative input (-1). I presume that this will fail during `time.sleep()`.

I know, that there is currently also no plausibility check for negative inputs on `reruns` but this is a topic for a different issue, as it is not failing but silently not collecting any tests.

Reply:
Not sure what you mean here. Should I some kind of `delay_time` validation to the plugin code, so that it will some how handle a negative value?

Reply:
That's exactly what I meant. I would prefere to the code to work as if there would be no `delay_time` set, in case of a negative delay time and give a warning.